lua/entities/turret_bullets/shared.lua

```diff
@@ -32,27 +32,30 @@ function ENT:DoShot()
         end
 
         if IsValid( self.shootPos ) and SERVER then
+            local bulletDamage = 400 * self.ShotInterval -- ensuring dps of 400
             self.shootPos:FireBullets( {
                 Num = 1,
                 Src = self.shootPos:GetPos() + self.shootPos:GetAngles():Forward() * 10,
                 Dir = self.shootPos:GetAngles():Forward() * 1,
                 Spread = Vector( 0.015, 0.015, 0 ),
                 Tracer = 0,
-                Force = 2,
-                Damage = 25,
+                Force = bulletDamage,

```

Comment:
If the damage gets too high, this Force will be come pretty crazy

Reply:
It is quite fun to shoot something and watch it fly across half of spawn, not gonna add a clamp, make a new func or anything because it would be even more complicated, will probably just replace it with a normal value next time it needs to be edited.